src/include/osl_pvt.h

```diff
@@ -629,6 +629,18 @@ class Symbol {
         m_lastwrite  = std::max (m_lastwrite, lw);
     }
 
+    // Mark the symbol as always being read (and, if write==true, also
+    // that it's always written). Then is for when we don't know when
```

Comment:
You're right. I think I had to read that about 10 times wondering what you were talking about before I saw the mistake.